doc/nrf/ug_bootloader_external_flash.rst

```diff
@@ -3,13 +3,57 @@
 Using external flash memory partitions
 ######################################
 
+
 When using MCUboot, you can store the storage partition for the secondary slot in the external flash memory, using a driver for the external flash memory that supports the following features:
 
 * Single-byte read and write.
 * Writing data from the internal flash memory to the external flash memory.
 
+.. note::
+
+   The Partition Manager will only support run-time access to flash partitions defined in regions placed on placed on external flash devices that have drivers compiled in.
+   Partition Manager can not determine which partitions will be used at runtime, it can only determine which have or have not drivers enabled, so it will decide to only include into, partition map, partitions on flash devices with drivers enabled.
```

Comment:
```suggestion
   Partition Manager cannot determine which partitions will be used at runtime, but only those that have drivers enabled; it will only include such partitions into the partition map.
```